.github/workflows/test.yml

```diff
@@ -7,205 +7,12 @@ on:
-
-      - name: Setup PHP
-        uses: shivammathur/setup-php@v2
-        with:
-          php-version: ${{ matrix.php-version }}
-
-      - name: Run coding standards checks
-        run: |
-          cd html
-          ./bin/phpcs -p ${LOCALGOV_DRUPAL_PROJECT_PATH}
-
-  phpstan:
-    name: Deprecated code checks
-    needs: build
-    runs-on: ubuntu-latest
-
-    strategy:
-      fail-fast: false
-      matrix:
-        localgov-version:
-          - '3.x'
-        drupal-version:
-          - '~10.0'
-        php-version:
-          - '8.1'
-          - '8.2'
-
-    steps:
-
-      - name: Cached workspace
-        uses: actions/cache@v2
-        with:
-          path: ./html
-          key: localgov-build-${{ matrix.localgov-version }}-${{ matrix.drupal-version }}-${{ matrix.php-version }}-${{ github.run_id }}-${{ secrets.CACHE_VERSION }}
-          restore-keys: |
-            localgov-build-${{ matrix.localgov-version }}-${{ matrix.drupal-version }}-${{ matrix.php-version }}-
-
-      - name: Setup PHP
-        uses: shivammathur/setup-php@v2
-        with:
-          php-version: ${{ matrix.php-version }}
-
-      - name: Run deprecated code checks
-        run: |
-          cd html
-          ./bin/phpstan analyse -c ./phpstan.neon ${LOCALGOV_DRUPAL_PROJECT_PATH}
-  phpunit:
-    name: PHPUnit tests
-    needs: build
-    runs-on: ubuntu-latest
-
-    strategy:
-      fail-fast: false
-      matrix:
-        localgov-version:
-          - '3.x'
-        drupal-version:
-          - '~10.0'
-        php-version:
-          - '8.1'
-          - '8.2'
-
-    steps:
-
-      - name: Clone Drupal container
-        uses: actions/checkout@v2
-        with:
-          repository: localgovdrupal/drupal-container
-          ref: php${{ matrix.php-version }}
-
-      - name: Cached workspace
-        uses: actions/cache@v2
-        with:
-          path: ./html
-          key: localgov-build-${{ matrix.localgov-version }}-${{ matrix.drupal-version }}-${{ matrix.php-version }}-${{ github.run_id }}-${{ secrets.CACHE_VERSION }}
-          restore-keys: |
-            localgov-build-${{ matrix.localgov-version }}-${{ matrix.drupal-version }}-${{ matrix.php-version }}-
-
-      - name: Start Docker environment
-        run: docker-compose -f docker-compose.yml up -d
-
-      - name: Run PHPUnit tests
-        run: |
-          mkdir -p ./html/web/sites/simpletest && chmod 777 ./html/web/sites/simpletest
-          sed -i "s#http://localgov.lndo.site#http://drupal#" ./html/phpunit.xml.dist
-          docker exec -t drupal bash -c 'chown docker:docker -R /var/www/html'
-          docker exec -u docker -t drupal bash -c "cd /var/www/html && ./bin/phpunit /var/www/html/${{ env.LOCALGOV_DRUPAL_PROJECT_PATH }}"
+  tests:
+    uses: localgovdrupal/localgov_shared_workflows/.github/workflows/test-module.yml@feature/1.x/override-phpstan-neon-path
```

Comment:
@ekes I think this needs to change to `localgovdrupal/localgov_shared_workflows/.github/workflows/test-module.yml@1.x` now it's merged on workflows.  @rupertj can you do a follow up PR to change this please

Reply:
Yup, I'm fixing all the things there. Will merge it in once I've got all the tests green again.